### DISCUSSION TITLE
Menu Entry Swap: Book of the Dead

### DISCUSSION THREAD

**Discussion by DrYoshiyahu** [runelite/runelite#15008] [Q&A] [answered]
It would be a good idea to include the Book of the Dead in the list of options for the Menu Entry Swapper. The default left click for Kharedst's Memoirs is 'Reminisce', which is the teleport action, but the default left click for its upgraded version, Book of the Dead, is 'Wield'. In some ways, this makes it a downgrade, because it's slightly less convenient to use. This seems like a perfect use-case for the plugin, especially for anyone who is very used to using Kharedst's Memoirs.

![image](https://user-images.githubusercontent.com/14756430/170604922-a15d9912-1c98-470f-a156-b50fb966c479.png)

![image](https://user-images.githubusercontent.com/14756430/170605053-2dbf2843-ad97-4111-b4b4-4a3cb03ffce5.png)

**Comment by geheur** [comment, marked as answer]
are you aware of the custom left-click and custom shift-click swaps for inventory items? it sounds like you could accomplish this with those features already.

**Reply by DrYoshiyahu** [reply]
Yeah, that's definitely true, and that's what I've done. I'm not entirely familiar with the reasoning behind the inclusion of certain items in the list of default preset options, so maybe this isn't one of those situations, where the item is so ubiquitous. 🤷‍♂️

**Reply by Enriath** [reply]
The options in the config pre-date the custom left click swap. At this point, we're not really going to add any more swaps like that when it can be done with the custom swaps.
